foundry.toml

```diff
@@ -3,9 +3,7 @@ src = "src"
 out = "out"
 libs = ["lib"]
 remappings = [
-    'aave-helpers/=lib/aave-helpers/src/',
-    'solidity-utils/=lib/aave-helpers/lib/solidity-utils/src/',
-    'aave-address-book/=lib/aave-helpers/lib/aave-address-book/src'
+    'xchain-helpers/=lib/xchain-helpers/src/'
```

Comment:
This should be picked up automatically without needing to manually specify the remapping.